client/src/pages/SearchResultsPage.jsx

```diff
@@ -1,14 +1,155 @@
+import { Stack, Typography } from "@mui/material";
 
 import BasePage from "./BasePage";
+import FiltersBar from "../components/FiltersBar";
 
 /**
  * https://github.com/BobaTalks/internship-tracker/issues/28
  * This page will display all the search results. Part of this work we should include the
  * search and location inputs.
  */
+
+const mockFilterData = {
+  remote: {
+    filterName: "Remote",
+    data: {
+      onsite: {
+        label: "On-site",
+        count: 4,
+        checked: false,
+      },
+      remote: {
+        label: "Remote",
+        count: 5,
+        checked: false,
+      },
+      hybrid: {
+        label: "Hybrid",
+        count: 12,
+        checked: false,
+      },
+    },
+  },
+  company: {
+    filterName: "Company",
+    data: {
+      meta: {
+        label: "Meta",
+        count: 4,
+        checked: false,
+      },
+      amazon: {
+        label: "Amazon",
+        count: 5,
+        checked: false,
+      },
+      netflix: {
+        label: "Netflix",
+        count: 15,
+        checked: false,
+      },
+      google: {
+        label: "Google",
+        count: 12,
+        checked: false,
+      },
+      apple: {
+        label: "Apple",
+        count: 2,
+        checked: false,
+      },
+    },
+  },
+  education: {
+    filterName: "Education",
+    data: {
+      currentStudent: {
+        label: "Current student",
+        count: 4,
+        checked: false,
+      },
+      recentGraduate: {
+        label: "Recent graduate",
+        count: 5,
+        checked: false,
+      },
+    },
+  },
+  semester: {
+    filterName: "Semester",
+    data: {
+      spring2023: {
+        label: "Spring 2023",
+        count: 4,
+        checked: false,
+      },
+      summer2023: {
+        label: "Summer 2023",
+        count: 5,
+        checked: false,
+      },
+      fall2023: {
+        label: "Fall 2023",
+        count: 10,
+        checked: false,
+      },
+      spring2024: {
+        label: "Spring 2024",
+        count: 23,
+        checked: false,
+      },
+      summer2024: {
+        label: "Summer 2024",
+        count: 8,
+        checked: false,
+      },
+    },
+  },
+  datePosted: {
+    filterName: "Date posted",
+    data: {
+      anyTime: {
+        label: "Any time",
+        count: 3,
+        checked: false,
+      },
+      pastDay: {
+        label: "Past 24 hours",
+        count: 2,
+        checked: false,
+      },
+      pastWeek: {
+        label: "Past week",
+        count: 21,
+        checked: false,
+      },
+      pastMonth: {
+        label: "Past month",
+        count: 12,
+        checked: false,
+      },
+    },
+  },
+};
+
+export const FilterContext = createContext([{}, () => {}]);
+
 const SearchResultsPage = () => {
-  return <BasePage></BasePage>;
+  const [filterData, setFilterData] = useState(mockFilterData);
+
+  return (
+    <FilterContext.Provider value={[filterData, setFilterData]}>
```

Comment:
One thing we want to consider maybe for later too, is persisting the filters for future use. If a user selects 5 different filter options, and then refreshes their browser, they'll lose the time the spent selecting those options to filter their results.

Are there any solutions that you can think of that would solve that?

Reply:
I've used local storage to solve this problem

Reply:
What if the user wants to share the filter with another student?

More just probing to think about some other possibilities, but I think local storage is a viable solution as well for certain use cases

Reply:
wdym by sharing filters? would there be a button to save the filter and share the link to someone else?
would you like me to keep the local storage for now?

Reply:
I think the hint is to use query parameters to set the filters on render/useEffect, such that if anyone had that link, they'd be able to set the filters in that way

Reply:
What Michael said! If we store filters as query params, users could also bookmark different combinations of filters if they wanted

Reply:
ohhh that's cool I don't think I've ever done that in this context before

Reply:
Is this done using useSearchParams hook? Should we put this in a new ticket?

Reply:
We can push it to a new ticket, too, local storage probably okay, but another concern we'd want to consider or ask: do we know the user wants to save everything on refresh?

We're forcing users to manually select each filter and clearing them now. Might be a question for design on what that experience should be

Reply:
That's true...I've thought about that as well. I'll ask the design team. Can we push this to a new ticket while we wait for their response?